modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldServiceImpl.java

```diff
@@ -103,6 +103,16 @@ public ObjectField getObjectField(long objectFieldId)
 		return objectFieldLocalService.getObjectField(objectFieldId);
 	}
 
+	@Override
+	public List<ObjectField> getObjectFields(long objectDefinitionId)
+		throws PortalException {
+
+		_objectDefinitionModelResourcePermission.check(
+			getPermissionChecker(), objectDefinitionId, ActionKeys.VIEW);
+
+		return objectFieldLocalService.getObjectFields(objectDefinitionId);
+	}
```

Comment:
@p-albuquerque are we using this method?

Reply:
I used this in the past way, but I was change the approach and forgot to undone this.